readme.md

```diff
@@ -48,7 +48,7 @@ createuser -s postgres # create general purpose postgres user
 brew install imagemagick # image-processing
 
 # project setup
-bin/setup
+script/setup
```

Comment:
I think this here was simply a typo. It seems to me `bin/setup` is simply not supported. At least not when the project is set up newly.